ui/oopsyraidsy/oopsyraidsy.js

```diff
@@ -117,6 +117,7 @@ let kLatePullText = {
 
 const kPartyWipeText = {
   en: 'Party Wipe',
+  de: 'Gruppe ausgelöscht',
```

Comment:
`fr: 'Party Wipe',`
Can you added this, plz.

Reply:
Is there a reason not to translate that into French? Or do people say Party Wipe in French? Can you open a pull request for this?